dist/angular-editable-text.js

```diff
@@ -26,13 +26,12 @@
         onChange: '&ggOnChange'
       },
       transclude: true,
+      // Updated template to allow ellipsis to work when displaying long names in IE
```

Comment:
I'll remove it.  We'll have this to refer to if we ever want to update the main library.